gemma-core/src/main/java/ubic/gemma/persistence/service/AbstractService.java

```diff
@@ -16,7 +22,11 @@
  */
 public abstract class AbstractService<O extends Identifiable> implements BaseService<O> {
 
-    protected static final Log log = LogFactory.getLog( TaxonServiceImpl.class );
+    /**
+     * @deprecated define your own logger
+     */
+    @Deprecated
+    protected static final Log log = LogFactory.getLog( AbstractService.class );
```

Comment:
This is confusing because we cannot know from which implementation the error occured.